packages/repository-tests/src/relations/fixtures/models/order.model.ts

```diff
@@ -12,8 +12,9 @@ export class Order extends Entity {
   @property({
     type: 'string',
     id: true,
+    generated: true,
   })
-  id: string;
+  id: number;
```

Comment:
Most of acceptance files use models/repos from `fixtures`. We were thinking to add them to test files individually so that we can set the type to `features.idType`. But that would be lots of repeating code.  
So we are using [this](https://github.com/strongloop/loopback-next/pull/3538/commits/9fa74e5abac210e7145d638c0ee09b036d587580#diff-70c77278b0038ca734ce5e04c92d8d24R44) as a workaround.
Do you have any suggestions to modify the types?